src/main/java/org/jenkinsci/plugins/pipeline/github/MilestoneGroovyObject.java

```diff
@@ -0,0 +1,74 @@
+package org.jenkinsci.plugins.pipeline.github;
+
+import groovy.lang.GroovyObjectSupport;
+import org.eclipse.egit.github.core.Milestone;
+import org.eclipse.egit.github.core.User;
+import org.jenkinsci.plugins.scriptsecurity.sandbox.whitelists.Whitelisted;
+
+import java.io.Serializable;
+import java.util.Date;
+
+/**
+ * Groovy object that represents a GitHub Milestone.
+ *
+ * @author jamespullar
+ */
+
+public class MilestoneGroovyObject extends GroovyObjectSupport implements Serializable {
```

Comment:
Since this implements `Serializable`, that implies all it's members are also `Serializable`.

I haven't checked, is `Milestone` `Serializable` too?